src/hatti/views/photos.cljs

```diff
@@ -91,6 +78,11 @@
                                 js/PhotoSwipeUI_Default
                                 (clj->js photos)
                                 options)]
+    (set! (.. gallery
+              -options
+              -getImageURLForShare)
+          (fn [share-button-data]
+            (or (aget gallery "currItem" "original-src") "")))
```

Comment:
Also, `src` attribute will usually have an empty string or a string value but not a null value. May be that's why the function returns a string value